README.md

```diff
@@ -8,6 +8,21 @@ It includes options to automatically batch all the `.heic` files it finds in a f
 
 [Download the latest release](https://github.com/danbrakeley/heic2png/releases), and then put the exe somewhere in your path.
 
+### Compiling on Linux
+
+1. Install golang
+  - Via apt `apt install golang-go`
```

Comment:
`apt` is distribution specific, and historically has really old versions of languages, so I'd rather not include it here at all.